doc/plugins/org.polarsys.capella.core.ui.intro/src/org/polarsys/capella/core/ui/intro/views/WelcomeView.java

```diff
@@ -0,0 +1,249 @@
+package org.polarsys.capella.core.ui.intro.views;
+
+import java.util.LinkedHashSet;
+import java.util.Set;
+
+import javax.inject.Inject;
+
+import org.eclipse.core.runtime.CoreException;
+import org.eclipse.core.runtime.IConfigurationElement;
+import org.eclipse.core.runtime.PerformanceStats;
+import org.eclipse.core.runtime.Platform;
+import org.eclipse.jface.resource.ImageDescriptor;
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.graphics.Font;
+import org.eclipse.swt.graphics.GC;
+import org.eclipse.swt.graphics.Image;
+import org.eclipse.swt.graphics.Rectangle;
+import org.eclipse.swt.layout.GridData;
+import org.eclipse.swt.layout.GridLayout;
+import org.eclipse.swt.widgets.Canvas;
+import org.eclipse.swt.widgets.Composite;
+import org.eclipse.swt.widgets.Label;
+import org.eclipse.ui.IMemento;
+import org.eclipse.ui.IWorkbench;
+import org.eclipse.ui.PartInitException;
+import org.eclipse.ui.forms.HyperlinkGroup;
+import org.eclipse.ui.forms.widgets.ImageHyperlink;
+import org.eclipse.ui.internal.intro.impl.IIntroConstants;
+import org.eclipse.ui.internal.intro.impl.IntroPlugin;
+import org.eclipse.ui.internal.intro.impl.model.loader.ContentProviderManager;
+import org.eclipse.ui.internal.intro.impl.util.Log;
+import org.eclipse.ui.intro.IIntroSite;
+import org.eclipse.ui.part.IntroPart;
+import org.eclipse.ui.plugin.AbstractUIPlugin;
+
+
+public class WelcomeView extends IntroPart{
+
+	/**
+	 * The ID of the view as specified by the extension.
+	 */
+	public static final String ID = "org.polarsys.capella.core.ui.intro.views.WelcomeView";
+	public static final String EXTENSION_POINT_ID = "org.polarsys.capella.core.ui.intro";
+	public static final String EXTENSION_POINT_ID2 = "org.eclipse.ui.intro.configExtension";	
+	
+	Image banner_icon = null;
+	Image compass_icon = null;
+	Font titleTextFont = null;
+	Font introTextFont = null;
+		
+	public WelcomeView() {
+		contributions = new LinkedHashSet<WelcomePageContribution>();
+	}
+	
+	@Override
+	public void init(IIntroSite site, IMemento memento)
+			throws PartInitException {
+		super.init(site, memento);
+		IntroPlugin.getDefault().closeLaunchBar();
+		// load the correct model based in the current Intro Part id. Set the
+		// IntroPartId in the manager class.
+	}
+	
+
+	@Inject IWorkbench workbench;
```

Comment:
if unused, can you remove the variable (and then the javax.inject dependecies on manifest